javascript/ql/lib/semmle/javascript/internal/CachedStages.qll

```diff
@@ -167,8 +171,8 @@ module Stages {
       or
       exists(any(DataFlow::PropRef ref).getBase())
       or
-      exists(any(DataFlow::ClassNode cls))
-      or
+      // exists(any(DataFlow::ClassNode cls)) // Depends on AnalyzedNode
```

Comment:
Given the "DON'T USE!" comment on this predicate, I assume it's a deliberate choice to leave this commented-out, but just mentioning it in case you had intended to come back and fix/uncomment it.